Base/Python/slicer/util.py

```diff
@@ -928,64 +927,131 @@ def saveScene(filename, properties={}):
 #
 
 def moduleSelector():
+  """Return module selector widget.
+  Throws a RuntimeError exception if there is no module selector (for example, the application runs without a main window).
+  :return: module widget object
+  """
   w = mainWindow()
   if not w:
-    import sys
-    print("Could not find main window", file=sys.stderr)
-    return None
+    raise RuntimeError("Could not find main window")
   return w.moduleSelector()
 
 def selectModule(module):
+  """Set currently active module.
+  Throws a RuntimeError exception in case of failure (no such module or the application runs without a main window).
+  :param module: module name or object
+  """
   moduleName = module
   if not isinstance(module, str):
     moduleName = module.name
   selector = moduleSelector()
   if not selector:
-    import sys
-    print("Could not find moduleSelector in the main window", file=sys.stderr)
-    return None
+    raise RuntimeError("Could not find moduleSelector in the main window")
   moduleSelector().selectModule(moduleName)
 
 def selectedModule():
+  """Return currently active module.
+  Throws a RuntimeError exception in case of failure (no such module or the application runs without a main window).
+  :return: module object
+  """
   selector = moduleSelector()
   if not selector:
-    import sys
-    print("Could not find moduleSelector in the main window", file=sys.stderr)
-    return None
+    raise RuntimeError("Could not find moduleSelector in the main window")
   return selector.selectedModule
 
 def moduleNames():
+  """Get list containing name of all successfully loaded modules.
+  :return: list of module names
+  """
   from slicer import app
   return app.moduleManager().factoryManager().loadedModuleNames()
 
 def getModule(moduleName):
+  """Get module object from module name.
+  Throws a RuntimeError exception in case of failure (no such module).
+  :return: module object
+  """
   from slicer import app
   module = app.moduleManager().module(moduleName)
   if not module:
-    import sys
-    print("Could not find module with name '%s" % moduleName, file=sys.stderr)
-    return None
+    raise RuntimeError("Could not find module with name '%s'" % moduleName)
   return module
 
 def getModuleGui(module):
+  """Deprecated. Use getModuleWidget function instead"""
```

Comment:
I suggest to use the 'deprecated directive.  See https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated